library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -153,18 +160,39 @@ class Client() {
         return Client(address = address, privateKeyBundleV1 = bundle, apiClient = apiClient)
     }
 
-    fun create(account: SigningKey, options: ClientOptions? = null): Client {
+    fun create(
+        account: SigningKey,
+        options: ClientOptions? = null,
+    ): Client {
         val clientOptions = options ?: ClientOptions()
         val apiClient =
             GRPCApiClient(environment = clientOptions.api.env, secure = clientOptions.api.isSecure)
-        return create(account = account, apiClient = apiClient, options = options)
+        return create(
+            account = account,
+            apiClient = apiClient,
+            options = options,
+        )
     }
 
-    fun create(account: SigningKey, apiClient: ApiClient, options: ClientOptions? = null): Client {
+    fun create(
+        account: SigningKey,
+        apiClient: ApiClient,
+        options: ClientOptions? = null,
+    ): Client {
         return runBlocking {
             try {
                 val privateKeyBundleV1 = loadOrCreateKeys(account, apiClient, options)
-                val client = Client(account.address, privateKeyBundleV1, apiClient)
+                val libXMTPClient: FfiXmtpClient =
+                    createClient(
+                        logger = logger,
+                        ffiInboxOwner = account,
+                        host = "https://dev.xmtp.network:5556",
```

Comment:
Steven is working on getting MLS endpoints deployed to dev right now, but until then I'd suggest localhost with is_secure set to false.

Reply:
I'm getting a `API error: setup error: transport error` now that I changed this to `10.0.2.2:5556` (android equivalent of localhost) Is that the correct endpoint I'm supposed to be pointing at?

Reply:
O wild it passes but only if I set isSecure to true 🤷‍♀️

Reply:
Interesting. Usually it fails hard connecting to localhost using TLS. Have you tried actually doing anything with the SDK. I don't think it makes any network requests during client instantiation.

Just to confirm, are you using `xmtp/node-go:mls` as your Docker image for the local node?